File: client/src/app/classes/borrower.ts

export class Borrower {
    constructor(id: number, tz: string, ageCategory: number,
        firstName: string, lastName: string, phoneNumber: string, email: string) {
        this.id = id;
        this.tz = tz;
        this.ageCategory = ageCategory;
        this.firstName = firstName;
        this.lastName = lastName;
        this.phoneNumber = phoneNumber;
        this.email = email;
    }
    id: number;
    tz: string;
    ageCategory: number;
    firstName: string;
    lastName: string;
    phoneNumber: string;
    email: string;
}

File: project/DAL/Borrower.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace DAL
{
    public class Borrower
    {
        public int Id { get; set; }
        public string Tz { get; set; }
        public string FirstName { get; set; }
        public string LastName { get; set; }
        public string PhoneNumber { get; set; }
        public string Email { get; set; }
        public int? IdCategory { get; set; }
       public virtual Category  Category { get; set; }
        public virtual List<Lendings>  Lendings { get; set; }

    }
}
